libc/src/__support/fixedvector.h

```diff
@@ -24,6 +24,16 @@ template <typename T, size_t CAPACITY> class FixedVector {
 public:
   constexpr FixedVector() = default;
 
+  template <typename It> constexpr FixedVector(It begin, It end) {
```

Comment:
I feel like this constructor template needs to be a bit more restrictive on the types.

Reply:
Restricted it to only cpp::array input iterators.